### PR TITLE
[Kaptain] COPS-7232 profile creation

### PR DESCRIPTION
**Pull request by gvasquezvargas**
## Jira Ticket
https://jira.d2iq.com/browse/COPS-7232

## Description of changes being made
Instruction on how to enable automatic creation of profiles was not working. Dev provided the correct command to fix this issue and documentation.
This is fixed on 2.0 already, so no need to include there too.
Also did some linting, you can ignore that and just review lines 46 to 60.

### Preview

http://docs-d2iq-com-pr-4421.s3-website-us-west-2.amazonaws.com/

### CODE REVIEW

File: pages/dkp/kaptain/1.3.0/user-management/index.md
Comment (alembiewski) [inline pages/dkp/kaptain/1.3.0/user-management/index.md:46]: We have this [blog post](https://d2iq.com/blog/what-you-need-to-know-about-cryptomining-attacks-on-kubeflow), which explains why we disable this feature by default. Maybe we should add a link here for more context?

File: pages/dkp/kaptain/1.3.0/user-management/index.md
Comment (alembiewski) [inline pages/dkp/kaptain/1.3.0/user-management/index.md:46]: I think it still might be a good idea to show how to enable it during the fresh installation. Should we keep this command in addition to `kubectl kudo update`?

**Review by alembiewski — APPROVED** [review]
Thanks, @gvasquezvargas! Overall LGTM, left a few minor suggestions

File: pages/dkp/kaptain/1.3.0/user-management/index.md
Comment (MessKon) [inline pages/dkp/kaptain/1.3.0/user-management/index.md:48]: not sure if we want to mention this but the exact command to run is
image
and can be found [on this page](https://docs.d2iq.com/dkp/kaptain/1.3.0/install/konvoy-dkp/).

**Review by MessKon — APPROVED** [review]
looks good! ✅ 
just one minor suggestion raised

File: pages/dkp/kaptain/1.3.0/user-management/index.md
Comment (gvasquezvargas) [inline pages/dkp/kaptain/1.3.0/user-management/index.md:46]:
```suggestion
When an authenticated user logs into the system and visits the central dashboard for the first time, they trigger a profile creation. This is referred to as a "Registration Flow". However, automatic profile creation is [disabled by default](https://d2iq.com/blog/what-you-need-to-know-about-cryptomining-attacks-on-kubeflow).
```

File: pages/dkp/kaptain/1.3.0/user-management/index.md
Comment (gvasquezvargas) [inline pages/dkp/kaptain/1.3.0/user-management/index.md:46]:
```suggestion
When an authenticated user logs into the system and visits the central dashboard for the first time, they trigger a profile creation. This is referred to as a "Registration Flow." However, automatic profile creation is [disabled by default](https://d2iq.com/blog/what-you-need-to-know-about-cryptomining-attacks-on-kubeflow).
```

**Review by mmcelroy91 — APPROVED** [review]

File: pages/dkp/kaptain/1.3.0/user-management/index.md
Comment (Colin1841) [inline pages/dkp/kaptain/1.3.0/user-management/index.md:46]:
```suggestion
When an authenticated user logs into the system and visits the central dashboard for the first time, they trigger a profile creation. This is referred to as a "Registration Flow." However, [for security reasons](https://d2iq.com/blog/what-you-need-to-know-about-cryptomining-attacks-on-kubeflow), automatic profile creation is disabled by default.
```

File: pages/dkp/kaptain/1.3.0/user-management/index.md
Comment (Colin1841) [inline pages/dkp/kaptain/1.3.0/user-management/index.md:48]: In these cases, even though the section is about profile creation, restate that here in place of "it".
We want to do this in case a user skips everything else and just goes to the commands, so they can still know for sure what "it" refers to.

File: pages/dkp/kaptain/1.3.0/user-management/index.md
Comment (Colin1841) [inline pages/dkp/kaptain/1.3.0/user-management/index.md:62]:
```suggestion
    After running this command, log out and back into the kubeflow dashboard. You are prompted to create a profile.
```

File: pages/dkp/kaptain/1.3.0/user-management/index.md
Comment (Colin1841) [inline pages/dkp/kaptain/1.3.0/user-management/index.md:65]: Replace "it" again here

**Review by Colin1841 — APPROVED** [review]

**Comment by gvasquezvargas** [comment]
Waiting on a short review by Tony before merging.

**Pull request merged**
